sdk/resourcemanager/azure-resourcemanager-keyvault/pom.xml

```diff
@@ -43,7 +43,10 @@
     <javaModulesSurefireArgLine>
       --add-opens com.azure.resourcemanager.authorization/com.azure.resourcemanager.authorization=ALL-UNNAMED
       --add-opens com.azure.resourcemanager.keyvault/com.azure.resourcemanager.keyvault=ALL-UNNAMED
+
+      --add-opens com.azure.core/com.azure.core.implementation.util=ALL-UNNAMED
     </javaModulesSurefireArgLine>
+    <revapi.skip>true</revapi.skip>
```

Comment:
what has breaked?

Reply:
Added it last time to suppress revapi error. Just checked without it and it passed... Will remove it.